spanner/transaction.go

```diff
@@ -1049,6 +1050,94 @@ func (t *ReadWriteTransaction) runInTransaction(ctx context.Context, f func(cont
 	return ts, nil
 }
 
+// ReadWriteStmtBasedTransaction provides a wrapper of ReadWriteTransaction in
+// order to run a read-write transaction in a statement-based way.
+//
+// This struct is returned by client.BeginReadWriteTransaction and contains
```

Comment:
nit: This should now be the following:

```suggestion
// This struct is returned by NewReadWriteStmtBasedTransaction and contains
```